resources/test-invoker.mjs

```diff
@@ -48,8 +67,51 @@ export class RAFTestInvoker extends TestInvoker {
     }
 }
 
+class AsyncRAFTestInvoker extends BaseRAFTestInvoker {
+    _scheduleCallbacks(resolve) {
+        requestAnimationFrame(async () => {
+            await this._syncCallback();
```

Comment:
_syncCallback and _asyncCallback should be rename to something else,  since, well, the patch makes _syncCallback rather asynchronous :)

Reply:
LOL, we had the same convo in a similar pr... what do you think about:

```
class TestInvoker {
    constructor(captureTest, captureLayout, reportResults) {
        this._captureTest = captureTest;
        this._captureLayout = captureLayout;
        this._reportResults = reportResults;
    }
} 
```